certified-connectors/Telnyx SMS/readme.md

```diff
@@ -0,0 +1,18 @@
+# Telnyx
+Telnyx Suite is a comprehensive cloud communication platform that provides communication capabilities via voice, SMS, fax, and IP services. Telnyx provides global high-speed connections for low latency, high uptime service. The Telnyx Suite includes SIP trunking, messaging API, voice API, lookup API, programmable fax API, and wireless services.
+
+## Publisher: ********
+
+## Prerequisites
+Sign-up free from https://telnyx.com/ to create an API key. For more information on creating the API keys are available from https://support.telnyx.com/en/articles/4305158-api-keys-and-how-to-use-them
+
+## Supported Operations
+### Send a message
+Send a message with a Phone Number, Alphanumeric Sender ID, Short Code or Number Pool.
+
+
+## API Documentation
+https://developers.telnyx.com/
+
+## Known Issues and Limitations
+None.
```

Comment:
@fabianp5060 please review the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1), all of the required headings should be present.

Reply:
resolved in commit 94d2e98